tests/maap_test_helper.py

```diff
@@ -12,6 +12,14 @@ def populate_maap_api_host(stage):
         env_prefix = f"{stage}."
     return f"api.{env_prefix}maap-project.org"
 
+def populate_maap_host(stage, prefix):
+    stage = stage if stage != "main" else "dit"
```

Comment:
Same question as above: where does a stage of `main` come from?

Reply:
This is pulling from the work Phil did, so the honest answer is that i'm not sure